lib/Controller/Library.php

```diff
@@ -1168,6 +1168,12 @@ public function add(Request $request, Response $response)
             $folderId = $this->getUser()->homeFolderId;
         }
 
+        $playlist = $this->playlistFactory->getById($parsedBody->getInt('playlistId'));
```

Comment:
If we're uploading media not on a playlist, then this parameter would be empty and I think throw a not found?

Reply:
@dasgarner Thank you for the review! I've updated the logic to address this, please see latest [commit ](https://github.com/xibosignage/xibo-cms/pull/2711/commits/da53951c8848138bdba4936f28bcd1cf28101c3e). Thanks again!